schedulers/terraform/managed-airflow-mwaa/main.tf

```diff
@@ -4,7 +4,9 @@
 
 module "mwaa" {
```

Comment:
move all mwaa resources to `mwaa.tf` file. 

Delete providers file and move provider code, locals and data sources to `main.tf`